sklearnex/__init__.py

```diff
@@ -1,5 +1,6 @@
 # ==============================================================================
 # Copyright 2021 Intel Corporation
+# Copyright contributors to the oneDAL project
```

Comment:
this brings interesting question. sklearnex is not part of UXL or oneDAL project - just opensource on it's own

Topic to discuss potentially within WG/UXL

Reply:
Thanks for your comment @napetrov. Will plan this discussion. 
Just wanted to know if Scikit-learn-intelex has any CLA process?

Till we reach a final conclusion on copyright, shall we mark this PR as draft?

Reply:
Update - 
Based on [LICENE](https://github.com/intel/scikit-learn-intelex/blob/main/LICENSE) file, I have modified the copyright text. 
@napetrov please review and suggest any changes.

Reply:
Looks good for now. I've started discussion on if we would like to change this in longerterm process. 

With one of the options to also give scikit to UXL or keeping things as is.  Will update on results of those discussions